cdm/misc/src/main/java/ucar/nc2/geotiff/GeoTiff.java

```diff
@@ -191,6 +192,30 @@ int writeData(byte[] data, int imageNumber) throws IOException {
     return nextOverflowData;
   }
 
+  int writeData(int[] data, int imageNumber) throws IOException {
```

Comment:
Untested at the moment

Reply:
Any suggestions for a test datafile that has integer variables? I found `problem/88129.nc`, but I can't seem to write the data back out properly (I need to investigate the test a little bit further on that).

Reply:
I completely forgot that my test file has an integer dtype in it. And utilizing this has revealed some other possible bugs that I'm now going to hunt down (possibly a bug with the test rather than the code).

Reply:
Figured out the test failure and there is a bug in the geotiff writer when doing lambert conformal and the y-axis goes from positive to negative. I'll open up a new PR for that.

Reply:
Opened https://github.com/Unidata/netcdf-java/pull/1342 to address this. I will need to rebase this PR to resolve the conflict since the test file I introduce here will be making an early appearance.